fe/fe-core/src/main/java/org/apache/doris/catalog/Database.java

```diff
@@ -556,6 +569,7 @@ public void write(DataOutput out) throws IOException {
         dbEncryptKey.write(out);
 
         out.writeLong(replicaQuotaSize);
+        dbProperties.put("transactionQuotaSize", String.valueOf(transactionQuotaSize));
```

Comment:
define a `static final string` for "transactionQuotaSize" key

Reply:
DONE